src/generic_ui/polymer/cloud-install.html

```diff
@@ -58,13 +63,39 @@
     <core-signals on-core-signal-open-cloud-install-dialog='{{ openLoginDialog }}'></core-signals>
 
     <!-- TODO: use core-pages rather than multiple dialogs -->
+    <uproxy-dialog layered='false' backdrop='true' id='regionDialog'>
```

Comment:
Right now the digitalocean provider only has one ```start``` method that encapsulates both logging in and creating the droplet (when the region is set).  I'd rather keep the provider this way, at least unless we get UX guidance that create separate steps really is important.  I'm inclined to think that there should just be one "Login and create droplet" button that the user should need to click (which would call ```start```) after the user selects all their options.